ajax/libs/cell/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "cell",
+  "filename": "cell.min.js",
+  "description": "A self-driving web app framework",
```

Comment:
Please change the description to this: `A self-constructing web app framework powered by a self-driving DOM.`